### PR TITLE
Fixes #19147: Email notification should be disabled when no host server is configured

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/19147

### CODE REVIEW

File: change-validation/src/main/resources/template/ChangeValidationManagement.html [hunk 1]
```diff
@@ -92,6 +92,35 @@
...
 
     <div class="inner-portlet">
+      <div class="page-title">Configure email notification</div>
+      <div class="portlet-content">
+        <div class="col-xs-12 callout-fade callout-warning">
+          <div class="marker">
+            <span class="glyphicon glyphicon-info-sign"></span>
+          </div>
+          <p>
+            By default, email notification are disabled. To enable it make sure that <b>smtp.hostServer</b> parameter is
```
Comment (Fdall) [inline change-validation/src/main/resources/template/ChangeValidationManagement.html:102]:
```suggestion
            By default, email notifications are disabled. To enable it make sure that the <b>smtp.hostServer</b> parameter is
```

File: change-validation/src/main/resources/template/ChangeValidationManagement.html [hunk 2]
```diff
@@ -92,6 +92,35 @@
...
+            not left empty in the configuration file: <b>/opt/rudder/etc/plugins/change-validation.conf</b>
+          </p>
+          <p>
+            By default, email notification are disabled. To enable it make sure that <b>smtp.hostServer</b> parameter is
```
Comment (Fdall) [inline change-validation/src/main/resources/template/ChangeValidationManagement.html:106]:
```suggestion
            By default, email notifications are disabled. To enable it make sure that the <b>smtp.hostServer</b> parameter is
```

**Review by Fdall — CHANGES_REQUESTED** [review]

File: change-validation/src/main/resources/template/ChangeValidationManagement.html (hunk 1 above)
Comment (ElaadF) [reply change-validation/src/main/resources/template/ChangeValidationManagement.html:102]: also : duplicated paragraph

**Comment by ElaadF** [comment]
Commit modified

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**